android/modules/cardview/.classpath

```diff
@@ -5,6 +5,6 @@
 	<classpathentry exported="true" kind="con" path="com.android.ide.eclipse.adt.DEPENDENCIES"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" path="gen"/>
-	<classpathentry exported="true" kind="lib" path="lib/cardview-v7-23.4.0.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/cardview-v7-23.0.1.jar"/>
```

Comment:
Yeah, that's from reverting https://github.com/appcelerator/titanium_mobile/pull/9027/commits/80b4302cbb84690591b32837f5dea9825ac4d4f1, `.classpath` aren't used anymore. They are left-overs from Eclipse.

Also, Jenkins is failing due to Android SDK 25 not being installed.